src/renderers/additional/tree-renderer.ts

```diff
@@ -139,15 +139,20 @@ export class TreeMasterDetailRenderer extends Renderer implements DataChangeList
 
     this.appendChild(div);
     this.dialog = document.createElement('dialog');
+    this.dialog.classList.add('jsf-treeMasterDetail-dialog');
     const title = document.createElement('label');
     title.innerText = 'Select the Item to create:';
+    title.classList.add('jsf-treeMasterDetail-dialog-title');
     this.dialog.appendChild(title);
     const dialogContent = document.createElement('div');
     dialogContent.classList.add('content');
+    dialogContent.classList.add('jsf-treeMasterDetail-dialog-content');
     this.dialog.appendChild(dialogContent);
     const dialogClose = document.createElement('button');
     dialogClose.innerText = 'Close';
     dialogClose.onclick = () => this.dialog.close();
+    dialogClose.classList.add('btn');
```

Comment:
This makes it compatible to materialize. However, that should be too specific for the general tree renderer. Should I change leave it, change it to `button` or remove it completely? I think having a general button class makes sense.